talaiot/src/main/kotlin/com/cdsap/talaiot/publisher/InfluxDbPublisher.kt

```diff
@@ -113,7 +113,12 @@ class InfluxDbPublisher(
         return report.tasks?.map { task ->
             Point.measurement(influxDbPublisherConfiguration.taskMetricName)
                 .time(System.currentTimeMillis(), TimeUnit.MILLISECONDS)
-                .tag(DefaultTaskDataProvider(task, report).get().filter { it.key != "value" } as Map<String, String>)
+                .tag(
+                    DefaultTaskDataProvider(task, report).get()
+                        .mapValues {
+                            it.value.toString()
+                        }
+                        .filter { it.key != "value" })
```

Comment:
Sorry, I missed because in my manual test I had task publishing disabled :(
Idea for future:  I wonder if it would be possible to cover that with a tests.

Reply:
np